### ISSUE TITLE
BlankLine does not work properly 

### ISSUE THREAD

**Issue by forgodtosave** [lezer-parser/python#14]
I compiled the python grammar locally and discovered the following issue: an blank line is not correctly recognized and therefore not properly skipped.
For example, if I make two blank lines in line 1 and 2, the whole program is interpreted as a blank line.
```


name = "two blanklines"
number = 20
```
![image](https://github.com/lezer-parser/python/assets/125274632/a34a5903-4d15-4975-be8c-2368f4743d63)

**Comment by marijnh** [comment]
Attached patch should help.

**Comment by forgodtosave** [comment]
I'm not sure if I did something wrong, but now the error appeaser even if only one blank line is in front of the text.
I think there is also 2 more bugs:
1)  multiple blank lines get merged into one, example:
```
name = "three blank lines"



name = ""
```
![image](https://github.com/lezer-parser/python/assets/125274632/7935610b-faef-438a-a124-6c2c55560c04)

2) the blank line rule has a problem with detecting the newline/space after blankLineStart, example: 
```
name = "blank line, two spaces"
  
name = "blank line, three spaces"
   
name=""
```
![image](https://github.com/lezer-parser/python/assets/125274632/1553acfd-7091-4a87-82c1-abcd3ba1a79b)

**Comment by marijnh** [comment]
The Python parser does not emit `BlankLine` as a node, so I'm somewhat at loss where these syntax trees you are showing are coming from.

**Comment by forgodtosave** [comment]
I changed the `BlankLine` rule to uppercase and also the only appearance in the skip statement. As far as I understand it should not change anything then the appearance in the syntax tree?

**Comment by marijnh** [comment]
I see zero error nodes in the output when parsing these pieces of code, so even if we account for the fact that you changed the name of blank line nodes, I am not getting the same output.

**Comment by forgodtosave** [comment]
That is strange. So a blank line in the first line is no longer a problem for you?

Even if I use `import {parser} from "@lezer/python"` the errors from 2. remain.
![image](https://github.com/lezer-parser/python/assets/125274632/4d3bc736-b5be-4b3c-b8fd-8d877c740f82)

**Comment by marijnh** [comment]
Could you just show the precise `parser.parse` call you're making that leads to these trees? For example [here](https://codemirror.net/try/?c=aW1wb3J0IHtwYXJzZXJ9IGZyb20gIkBsZXplci9weXRob24iCgpjb25zb2xlLmxvZyhwYXJzZXIucGFyc2UoYG5hbWUgPSAidGhyZWUgYmxhbmsgbGluZXMiCgoKCm5hbWUgPSB4YCkgKyAiIik=).

**Comment by forgodtosave** [comment]
I tried another approach to judge whether [here](https://codemirror.net/try/?c=aW1wb3J0IHtwYXJzZXJ9IGZyb20gIkBsZXplci9weXRob24iCgpjb25zb2xlLmxvZyhwYXJzZXIucGFyc2UoYG5hbWUgPSAidGhyZWUgYmxhbmsgbGluZXMiCgoKCm5hbWUgPSAiYmxhbmsgbGluZSwgemVybyBzcGFjZXMiCgpuYW1lID0gImJsYW5rIGxpbmUsIG9uZSBzcGFjZXMiCiAKbmFtZSA9ICJibGFuayBsaW5lLCB0d28gc3BhY2VzIgogIApuYW1lPSIiYCkgKyAiIik=) really is an error like in the syntax tree or not like in the log.
I have implemented the example codemirror linter in the project with the change to highlight `node.type.isError` and now you can see an error with every blank line. 
![image](https://github.com/lezer-parser/python/assets/125274632/6d0a327a-cec9-45f0-9ed6-7dc92bbb2497) 
(same input as above) 

It always seems to be the first character after blankLineStart that causes problems, since even with two sapces only the first space is marked. However, this behavior cannot be seen with several newlines

**Comment by forgodtosave** [comment]
I found a [solution for me](https://github.com/forgodtosave/mainsail/tree/KlipperConfig-Language/src/plugins/CodemirrorLanguages/KlipperConfigLanguage/parser).
I just let the external tokenizer swallow the spaces and then also the newline at the end. So I removed the whole blankLine rule from the grammar. Here the Comment in the blankLine rule becomes problematic. I solved this simply by allowing an optional newline after a comment. 
I hope I could help.
